content/en/docs/Configuration/Certificate Management/mtls.md

```diff
@@ -49,15 +51,16 @@ Note : The target architecture is that API-Gateway must route the request to ACP
 First, some certificates must exist to generates multiples
 
 ```bash
-openssl genrsa -out ca1.key 2048openssl req -new -x509 -days 3650 -key ca1.key -subj "/C=BR/ST=São Paulo/L=São Paulo/O=Axway/CN=Axway Root CA" -out ca1.crtopenssl genrsa -out ca2.key 2048openssl req -new -x509 -days 3650 -key ca2.key -subj "/C=BR/ST=São Paulo/L=São Paulo/O=Axway/CN=Axway Root CA" -out ca2.crt
+openssl genrsa -out ca1.key 2048openssl req -new -x509 -days 3650 -key ca1.key -subj "/C=BR/ST=São Paulo/L=São Paulo/O=Axway/CN=Axway Root CA" -out ca1.crtopenssl genrsa -out ca2.key 2048openssl req -new -x509 -days 3650 -key ca2.key -subj "/C=BR/ST=São Paulo/L=São Paulo/O=Axway/CN=Axway Root CA" -out ca2.crt
 ```
 
-### Create certificates for the Third Party Provider App (Client Certificates for each TPP) 
+### Create certificates for the Third Party Provider (TPP) App (Client Certificates for each TPP)
 
-Each certificate must have one key and signed with a root CA previously created. These configuration files below are provided as example.
+Each certificate must have one key and signed with a root CA previously created. These configuration files below are provided as examples.
```

Comment:
Each certificate must have one key that is signed with a previously created root certificate authority.